spec/spec_helper.rb

```diff
@@ -13,14 +13,18 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+require 'securerandom'
 require 'simplecov'
 require 'simplecov-cobertura'
 SimpleCov.formatters = [
   SimpleCov::Formatter::HTMLFormatter,
   SimpleCov::Formatter::CoberturaFormatter,
 ]
 
-SimpleCov.command_name('RSpec')
+# Give each coverage report a unique ID if we are running parallel tests
+if ENV['TEST_ENV_NUMBER']
```

Comment:
could we maintain the original behaviour if there aren't running parallel tests?
```
if ENV['TEST_ENV_NUMBER']
  SimpleCov.command_name("RSpec-#{SecureRandom.hex(4)}")
else
  SimpleCov.command_name('RSpec')
end
```